src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -13,6 +13,11 @@
      * <p>Это задание необходимо решать с использованием логических операторов.</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+         return (!(a & b) & !(a^b) & c & d) |
```

Comment:
some of checks were failed 
![image](https://user-images.githubusercontent.com/36365171/70029550-0aedb780-15b0-11ea-9522-8b3e5e659a01.png)